static/app/views/projectInstall/createProject.tsx

```diff
@@ -245,14 +245,19 @@ function CreateProject() {
           <FormLabel>{t('Team')}</FormLabel>
           <TeamSelectInput>
             <TeamSelector
+              data-test-id="select-team"
```

Comment:
it doesn't seem like this is used?

Reply:
oops. Yeah I ended up finding it by the typeahead